core/modules/proto/FakeProtocolFixture.h

```diff
@@ -74,8 +73,8 @@ class FakeProtocolFixture {
         }
```

Comment:
`s = f.mutable_subchunks();` repeats for both if and else, can be moved outside if, and merged with declaration.